calico-cloud.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Calico Cloud
+description: Learn about Calico Cloud.
+canonical_url: '/calico-cloud/index'
+---
+
+![calico-cloud]({{site.baseurl}}/images/calico-cloud-small.png)
+
+## If you could solve these 3 problems today...
+
+#### Workload access controls
+
+Implement pod-level workload access controls and protect containerized environments in the Kubernetes cluster from outside threats, while enabling applications and workloads to securely communicate with resources outside the cluster behind a firewall or other control point.
+
+#### Compliance
+
+Create compliance policies for any compliance framework including PCI, SOC 2, and more. Continuously monitor compliance for your container, Kubernetes, and cloud environment. Retain a daily history of your compliance status. Generate on-demand or scheduled audit reports.
+
+#### Faster troubleshooting
+
+Enable faster troubleshooting of Kubernetes workloads and applications with Service Graph, Packet Capture, anomaly detection, and performance hotspots, leading to shorter time-to-resolution, less application downtime, and improved quality of service
+
+## then take your cluster into Calico Cloud and see for yourself!
```

Comment:
In my opinion splitting this sentence in half and having the second half as a title doesn't look very pleasing visually, but if PM likes it OK

Reply:
I could not see it until now...could not build locally. I will find a different way to deal with it.

Reply:
Fixed